includes/admin/reporting/export/export-actions.php

```diff
@@ -177,6 +177,29 @@ function edd_include_file_downloads_batch_processer( $class ) {
 }
 
 /**
+ * Register the sales batch exporter
+ *
+ * @since  2.7
+ */
+function edd_register_sales_export_batch_export() {
+	add_action( 'edd_batch_export_class_include', 'edd_include_sales_export_batch_processer', 10, 1 );
+}
+add_action( 'edd_register_batch_exporter', 'edd_register_sales_export_batch_export', 10 );
```

Comment:
Does it matter? Is there a case you've thought of where the load order for export-specific classes would fail?